src/js/helpers/storage_init.js

```diff
@@ -0,0 +1,34 @@
+var storage_module = null;
+var local_storage = require('./local_storage'),
+	session_storage = require('./session_storage'),
+	single_cookie = require('./single_cookie'),
+	cookies       = require('./cookies');
+
+module.exports = {
+	validateType: function( storage_type ) {
+		// Default to valid_values[0] if storage_type is not in valid_values
+		var valid_values = ['cookies', 'singleCookie', 'localStorage', 'sessionStorage'];
+		return valid_values.indexOf( storage_type ) > -1 ? storage_type : valid_values[0];
```

Comment:
We can use array `includes` here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes. Also, I think it is better to be explicit and use `'cookies'` instead of `valid_values[0]`.

```suggestion
		return valid_values.includes( storage_type ) ? storage_type : 'cookies';
```